base/test_views.py

```diff
@@ -132,3 +132,16 @@ def test_redirect_servicios_escolares(self):
         self.client.login(username=self.username, password=self.password)
         response = self.client.get(reverse('home'))
         self.assertRedirects(response, reverse('becas:services'))
+
+    def test_redirect_login(self):
+        """ Test that a logged user gets redirected to home.
+
+        """
+        self.create_group(CAPTURISTA_GROUP)
+        self.capturista = Capturista.objects.create(user=self.user)
+        self.create_group(CAPTURISTA_GROUP)
+        self.client.login(username=self.username, password=self.password)
+        response = self.client.get(reverse('home'))
+        self.assertRedirects(response, reverse('captura:estudios'))
+        response = self.client.get(reverse('tosp_auth:login'))
+        self.assertTrue(response, reverse('home'))
```

Comment:
Assert true only takes a boolean expression as an argument, and here you are giving it two arguments.